src/typings.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'ethereumjs-tx';
```

Comment:
Rename this file to `global.d.ts`

Reply:
And add a TODO comment to flesh out the type interfaces we use from this package.